Server/src/controllers/review.controller.js

```diff
@@ -0,0 +1,88 @@
+import mentors from '../models/mentor.model';
+import users from '../models/users.model';
+import session from '../models/mentorship.model';
+import review from '../models/review.model';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+class ReviewController {
+  /**
+  * Review mentor after mentorship session
+  * @param {object} req
+  * @param {object} res
+  */
+  static reviewMentor(req, res) {
+    const isMentorshipSession = session.find(s => s.sessionId == parseInt(req.params.sessionId));
+    const isMentee = users.find(z => z.userId === parseInt(req.user.userId));
+    const isMenteeWhoReq = session.find(s => s.menteeId === parseInt(req.user.userId));
+
+    
+      if (!isMentorshipSession) {
+        return res.status(404).json({
+          status: 404,
+          error: 'this mentorship session not exist'
+        });
+      }      
+      
+      if (isMentee) {
+        if (isMenteeWhoReq.sessionId !== parseInt(req.params.sessionId) || isMenteeWhoReq.status !== 'accepted') {
+          return res.status(403).json({
+            status: 403,
+            error: "You did not requested this mentorship or not accepted already! "
+          });
+        } else {
+          const newReview = {
+            sessionId: isMenteeWhoReq.sessionId,
+            mentorId: isMenteeWhoReq.mentorId,
+            menteeId: isMenteeWhoReq.menteeId,
+            score: req.body.score,
+            menteeFullName: isMentee.firstName + ' ' + isMentee.lastName,
+            remark: req.body.remark
```

Comment:
make use of object destructuring
` const { score, remark, reason } = req.body `